tee-worker/omni-executor/aa-contracts/test/ValidationTests.t.sol

```diff
@@ -0,0 +1,697 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.13;
+
+import {Test, console} from "forge-std/Test.sol";
+import {OmniAccount} from "../src/accounts/OmniAccount.sol";
+import {OmniAccountTestable} from "./OmniAccountTestable.sol";
+import {EntryPoint} from "../src/core/EntryPoint.sol";
+import {UserOpSigner} from "../src/interfaces/UserOpSigner.sol";
+import {Passkey} from "../src/interfaces/Passkey.sol";
+import {Counter} from "../src/Counter.sol";
+import {OmniAccountTestUtils} from "./OmniAccountTestUtils.sol";
+import {TestUtils} from "./TestUtils.sol";
+import {PackedUserOperation} from "../src/interfaces/PackedUserOperation.sol";
+import {SIG_VALIDATION_FAILED, SIG_VALIDATION_SUCCESS} from "../src/core/Helpers.sol";
+import "@openzeppelin/contracts/utils/Base64.sol";
+
+contract ValidationTests is Test {
```

Comment:
Can we have `OmniAccountAsPassKey.t.sol`  ? We've got similiar for `Owner` and `Root`